enterprise/server/auth/auth.go

```diff
@@ -833,11 +838,13 @@ func (a *OpenIDAuthenticator) authenticateUser(w http.ResponseWriter, r *http.Re
 	// If the session is not found, bail.
 	sesh, err := authDB.ReadSession(ctx, sessionID)
 	if err != nil {
-		return nil, ut, status.PermissionDeniedError(loggedOutMsg)
+		log.Debugf("Session not found: %s", err.Error())
```

Comment:
nit: %s understands errors so there's no need to call .Error()

Reply:
Done